src/olympia/conf/prod/settings.py

```diff
@@ -35,14 +35,14 @@
 
 DATABASES = {
     'default': get_db_config('DATABASES_DEFAULT_URL', charset='utf8mb4'),
-    'slave': get_db_config(
-        'DATABASES_SLAVE_URL', atomic_requests=False, charset='utf8mb4',
+    'replica': get_db_config(
+        'DATABASES_REPLICA_URL', atomic_requests=False, charset='utf8mb4',
```

Comment:
note for later: we will have to co-ordinate with ops to update the env variables too. (Copying eviljeff comment from below.)